localized/games/eggquiz/eggquizmath_levels_ur.tsv

```diff
@@ -161,13 +161,13 @@ MiniTest_7_2	1	72001, 72002, 72003, 72004, 72005
 			72002	operations_drag	11~19+11~19=?	regroupingfalse	horizontal			20~30				
 			72003	operations_drag	20-1~10=?		horizontal			1~10				
 			72004	compare_number_magnitudes	descending	اعداد کو بڑے سے چھوٹے کی ترتیب دیں	सबसे बड़े से लेकर सबसे छोटे तक के क्रम में संख्या को रखे			51~99				
-			72005	word_problem	بلی كے پاس 2 مربلیس ہیں . اینڈرے كے پاس بلی سے 3 زیادہ مربلیس ہیں . اینڈرے كے پاس کل کتنے مربلیس ہیں ؟	बिली के पास two मार्बल्स हैं। ऑड्रे के पास बिली की तुलना में three अधिक मार्बल्स हैं। ऑड्रे के पास कितने मार्बल्स हैं?				5				
+			72005	word_problem	بختیار كے پاس 2 مربلیس ہیں . علیزہ كے پاس بختیار سے 3 زیادہ مربلیس ہیں . علیزہ كے پاس کل کتنے مربلیس ہیں ؟	बख़्तियार के पास two मार्बल्स हैं। अलीज़ा के पास बख़्तियार की तुलना में three अधिक मार्बल्स हैं। अलीज़ा के पास कितने मार्बल्स हैं?				5				
 MiniTest_7_3	1	73001, 73002, 73003, 73004, 73005												
```

Comment:
replace two, three, four, etc with 2, 3, 4, etc respectively in TTS speech statements.. Check line 164 and 170